package.json

```
@@ -142,7 +142,7 @@
     "react-coin-icon": "^0.1.36",
     "react-fast-compare": "^2.0.4",
     "react-flatten-children": "^1.1.2",
-    "react-native": "0.63.3",
+    "react-native": "0.63.3 ",
```

```suggestion
    "react-native": "0.63.3",
```

This will also break `package.json`